SingularityUI/app/views/tail.coffee

```diff
@@ -29,6 +29,12 @@ class TailView extends View
         @listenTo @collection, 'sync', =>
             @$el.removeClass 'fetching-data'
 
+        @listenTo @model, 'change', @setTaskHistory
```

Comment:
this is stretching my backbone knowledge, but is it possible to do something like this?

```coffeescript
@listenTo @model, 'change:isStillRunning', => @stopTailing() unless @model.get 'isStillRunning'
```